build/fix_tarballs_macos.sh

```diff
@@ -10,26 +10,29 @@ for tarball_dirty in $(find dist -regex ".*darwin.*_dirty\.\(tar.gz\)");do
   tarball=${tarball_dirty:5:${#tarball_dirty}-(5+13)} # Strips begining and end chars
   tarball="${tarball}.tar.gz"
   tarballTmpPath="dist/tarball_temp"
-  tarballContentPath="${tarballTmpPath}/newrelic-infra"
+  tarballContentPath="${tarballTmpPath}"
 
-  mkdir -p ${tarballContentPath}/etc/newrelic-infra/integrations.d
-  mkdir -p ${tarballContentPath}/usr/local/bin
-  mkdir -p ${tarballContentPath}/var/{db/newrelic-infra,log/newrelic-infra,run/newrelic-infra}
-  mkdir -p ${tarballContentPath}/var/db/newrelic-infra/{custom-integrations,integrations.d,newrelic-integrations}
+  echo "===> Create ${tarballContentPath}/Applications/newrelic-infra/ for binaries & licence"
```

Comment:
creating a var for `${tarballContentPath}/Applications/newrelic-infra` would save a lot of dups